screens/MenteeDetailsView.js

```diff
@@ -15,9 +16,41 @@ class MenteeDetailsView extends Component {
     headerTitleStyle
   });
 
+  state = {
+    messageDeltaString: '0 days'
+  };
+
+  async componentDidMount() {
+    try {
+      const messageContextJson = await AsyncStorage.getItem('messages');
+      if (messageContextJson) {
+        const messageContext = JSON.parse(messageContextJson);
+        const menteeId = this.props.navigation.state.params.mentee._id;
+        if (messageContext && menteeId in messageContext) {
+          const messageArray = messageContext[this.props.navigation.state.params.mentee._id];
+          if (messageArray) {
+            const messageDelta = Math.round(Math.abs(Date.now() - Date.parse(messageArray[0].createdAt)) / 86400000);
```

Comment:
Should we make 86400000 into a const to improve clarity, like `DAY_IN_MS`?

Reply:
Good call, fixing in next push